src/main/java/Storage.java

```diff
@@ -0,0 +1,97 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Storage class responsible for loading and saving data into hard drive.
+ */
+public class Storage {
+
+    private File hardDrive;
+
+    /**
+     * Constructor method for storage class.
+     * @param filePath file path for the hard drive file.
+     * @throws IOException - exception thrown when there is error in file path.
+     */
+    public Storage(String filePath) throws IOException {
+        this.hardDrive = new File(filePath);
+        if (hardDrive.createNewFile()) {
+            System.out.println("Hard Disk created.");
+        } else {
+            System.out.println("Hard Disk loaded.");
+        }
+    }
+
+    /**
+     * Loads hard drive from the file
+     * @throws FileNotFoundException if file is not found at target location.
+     */
+    public List<Task> load() throws IOException {
+        Scanner sc = new Scanner(this.hardDrive);
+        List<Task> tasks = new ArrayList<>();
+        while (sc.hasNextLine()) {
+            String[] entry = sc.nextLine().split(" / ");
+            switch (entry[0]) {
+            case "T":
+                tasks.add(new Todo(entry[2], Boolean.parseBoolean(entry[1])));
```

Comment:
Perhaps `entry[2]` should be declared as a variable with a descriptive name for better code readability?